sdk/servicebus/service-bus/test/utils/testUtils.ts

```diff
@@ -67,13 +68,13 @@ export class TestMessage {
     useSessions?: boolean,
     usePartitions?: boolean
   ): void {
```

Comment:
Update the `checkMessageContents` method with an additional check for the newly added `userId`.